pkgs/tools/audio/picotts/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, libtool, popt }:
+
+stdenv.mkDerivation {
+  name = "picotts-2018-10-19";
+  src = fetchFromGitHub {
+    repo = "picotts";
+    owner = "naggety";
+    rev = "2f86050dc5da9ab68fc61510b594d8e6975c4d2d";
+    sha256 = "1k2mdv9llkh77jr4qr68yf0zgjqk87np35fgfmnc3rpdp538sccl";
+  };
+  buildInputs = [ autoconf automake libtool popt ];
```

I suspect `autoconf` and `automake` are only needed at compile time, so they can be moved to `nativeBuildInputs`.
```suggestion
  nativeBuildInputs = [ autoconf automake ];
  buildInputs = [ libtool popt ];
```